R/getTemplate.R

```diff
@@ -20,26 +30,18 @@ getTemplate <- function(project = NULL) {
   filename <- if (project %in% names(templates)) templates[project] else project
   if (file.exists(filename)) {
     data <- read.csv2(filename, header = TRUE, sep = ";", na.strings = list(""), strip.white = TRUE, quote = "")
-    if (ncol(data) == 5) {  # is a mapping template generated by generateMappingfile
-      if ("Variable" %in% colnames(data) && ! "Unit" %in% colnames(data)) {
-        varunit <- magclass::unitsplit(data$Variable)
-        data$Variable <- varunit$variable
-        data$Unit <- varunit$unit
-      }
-      if ("piam_variable" %in% colnames(data) && ! "piam_unit" %in% colnames(data)) {
-        varunit <- magclass::unitsplit(data$piam_variable)
-        data$piam_variable <- varunit$variable
-        data$piam_unit <- varunit$unit
-      }
-    }
 
-    if (!all(c(
```

Comment:
I wonder whether for backward compatibility, we should keep a `colnames(data) <- tolower(colnames(data))`?

Reply:
Unless you have a concrete, realistic use case where we need backwards-compatibility here, I say no.

Reply:
No, I don't :)